includes/class-freemius.php

```diff
@@ -594,11 +649,13 @@ function _add_deactivation_feedback_dialog_box() {
 		 * @return array The uninstall reasons for the specified user type.
 		 */
 		function _get_uninstall_reasons( $user_type = 'long-term' ) {
+			$plugin_type = ( $this->_plugin->is_plugin() ? 'plugin' : 'theme' );
```

Comment:
@fajardoleo I would add a helper function to the plugin entity class and use it instead:

`function get_type() {
    return $this->is_plugin() ? 'plugin' : 'theme';
}
`